src/viewer.js

```diff
@@ -116,6 +116,29 @@ module.exports = class Viewer {
     this.gridHelper = null;
     this.axesHelper = null;
 
+    // https://stackoverflow.com/questions/16226693/three-js-show-world-coordinate-axes-in-corner-of-scene 
+
+    this.axisDiv = document.createElement('div');
+    this.el.appendChild( this.axisDiv );
+    this.axisDiv.classList.add('axis');
+
+    this.axisScene = new THREE.Scene();
+    this.axisCamera = new THREE.PerspectiveCamera( fov, this.axisDiv.clientWidth / this.axisDiv.clientHeight, 0.01, 10000 );
+    this.axisScene.add( this.axisCamera );
+
+    this.axisRenderer = new THREE.WebGLRenderer( { alpha: true } );
+    this.axisRenderer.physicallyCorrectLights = true;
+    this.axisRenderer.gammaOutput = true;
+    this.axisRenderer.gammaFactor = 2.2;
+    this.axisRenderer.setPixelRatio( window.devicePixelRatio );
+    this.axisRenderer.setSize( this.axisDiv.clientWidth, this.axisDiv.clientHeight );
+
+    this.axisCamera.up = this.defaultCamera.up;
+    this.axisCorner = new THREE.AxesHelper(5);
+    this.axisScene.add( this.axisCorner );
+
+    this.axisDiv.appendChild(this.axisRenderer.domElement);
```

Comment:
Could you move this block to a new method, `addAxisRenderer()`?

Reply:
Done